### PR TITLE
BarTemperatureLoad Implementation

### LINKED ISSUE #141: SAP2000_Toolkit: Add BarTemperatureLoad
#### Description:
Add functionality for handling BarTemperatureLoad

### PR DESCRIPTION
**Pull request by jean-lucjacksonBH**
### Issues addressed by this PR

Closes #141 

### Test files

https://burohappold.sharepoint.com/:f:/r/sites/BHoM/02_Current/12_Scripts/01_Test%20Scripts/SAP2000_Toolkit/141%20-%20Issue%20-%20BarTemperatureLoad?csf=1&web=1&e=IWVUdc

### CODE REVIEW

**Comment by FraserGreenroyd** [comment]
@jean-lucjacksonBH @JosefTaylor can we add labels to categorise this PR please :smile:

File: SAP2000_Adapter/CRUD/Create/Loads.cs
Comment (JosefTaylor) [inline SAP2000_Adapter/CRUD/Create/Loads.cs:440]: should be able to leave off replaceNow - only necessary for looping over directions when you've got a 3d load.

**Review by JosefTaylor — CHANGES_REQUESTED** [review]
Looks good. you can simplify the create a little by taking out replaceNow and just using replace- that's for looping over the directions in a 3D load.

**Review by JosefTaylor — APPROVED** [review]

**Comment by JosefTaylor** [comment]
The changes look good- needs a proper test though.

**Comment by maryannewachter** [comment]
Fixed AdapterIdReferences and checked with test files. Works as expected.

**Review by maryannewachter — APPROVED** [review]

**Comment by IsakNaslundBh** [comment]
Do not think we will be able to merge this in after all the merges that have happened:
https://github.com/BHoM/SAP2000_Toolkit/network

image

Think the things going on in there are a bit to much to handle with a manual rebase and conflict fixing, also looking at the changed files, there are multiple flagging up there as additions that already are on the master.

Would suggest to start this over from scratch in a github sense, that is, start from a new branch from master and add in any code written in here to that branch. Manually, not by merging this PR branch into it or similar, cause big risk it could lead to similar situation.

**Comment by JosefTaylor** [comment]
Closing this in favor of #160

**Pull request closed without merging**
